USAGE.md

```diff
@@ -0,0 +1,307 @@
+You need to capture the images of app view manually
+if you use custom view or 3rd party framework like a [Unity](https://unity3d.com/jp).
+
+Import USERDIVE iOS SDK header.
+
+```objective-c
+#import "Userdive.h"
+```
+
+Insert `[Userdive startDeveloperMode:<YOUR_TEAM_ID>]` into 
+`AppDeletegate:application:didFinishLaunchingWithOptions:` with your Team ID.
+
+```objective-c
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
+{
+  [Userdive setLogEnabled:YES];
+
+  // START DEVELOPER MODE WITHOUT TRACKERS
+  [Userdive startDeveloperMode:<your team id> trackers:nil];
+
+  return YES;
+}
+```
+
+When you want to capture view is displayed, call the capture method.
+
+Call `updateScreen` to prepare for capture the view.
+When you call `updateScreen` again, upload the captured image (You need to call `updateScreen` twice).
+
+```objective-c
+- (void)viewDidAppear:(BOOL)animated
+{
+  [super viewDidAppear:animated];
+
+  [Userdive updateScreen];
+  [Userdive setScreenName:@"screen A"];
+}
+```
+
+
+## Tracking gestures
+
+### Automatically detect view transitions
+
+USERDIVE iOS SDK can track user gesture and view transitions automatically.
+To start tracking, call `startTrackingMode:` when initializing SDK.
+
+Import USERDIVE iOS SDK header.
+
+```objective-c
+#import "Userdive.h"
+```
+
+Insert `[Userdive startTrackingMode:<YOUR_TEAM_ID>]` 
+`AppDeletegate:application:didFinishLaunchingWithOptions:` with your Team ID.
+
+```objective-c
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
+{
+    [Userdive setLogEnabled:YES];
+    [Userdive startTrackingMode:<your team id>];  // <-- HERE
+
+    return YES;
+}
+```
+
+### Manually view transitions
+
+In situations that have adopted a special view configuration
+(non-standard view placement and third-party libraries, etc.), 
+you might think that you want to switch the view tracking manually.
+
+Import USERDIVE iOS SDK header.
+
+```objective-c
+#import "Userdive.h"
+```
+
+Insert `[Userdive startTrackingMode:<YOUR_TEAM_ID> trackers:nil]` into 
+`AppDeletegate:application:didFinishLaunchingWithOptions:` with your Team ID.
+
+```objective-c
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
+{
+  [Userdive setLogEnabled:YES];
+
+  // START TRACKING MODE WITHOUT TRACKERS
+  [Userdive startTrackingMode:<your team id> trackers:nil];
+
+  return YES;
+}
+```
+
+Call `updateScreen` to change tracking view instance.
+And call `setScreenName` to set name of view.
+
+```objective-c
+- (void)viewDidAppear:(BOOL)animated
+{
+  [super viewDidAppear:animated];
+
+  [Userdive updateScreen];
+  [Userdive setScreenName:@"screen A"];
+}
+```
+
+
+## Using custom parameters
+
+You can set custom parameter for filterting on analytics view.
+For instance, customer age, gender, living area ... etc.
+
+Userdive can be accept custom parameters up to 5.
```

Comment:
Ellie please advice. How's below?
USERIDVE can adopt up to 5 custom parameters.